docs/Installation-Guide.rst

```diff
@@ -558,6 +558,35 @@ Docker
 
 Refer to `GPU Docker folder <https://github.com/microsoft/LightGBM/tree/master/docker/gpu>`__.
 
+Build CUDA Version (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `original GPU version <#build-gpu-version>`__ of LightGBM is based on OpenCL. This version is a separate implementation and requires NVIDIA graphics card with compute capability 6.0 and higher.
+
+This version is very experimental and we suggest using it only when it is impossible to use OpenCL version, for instance, on IBM POWER microprocessors.
+
+**Note**: only Linux is supported, other operating systems are not supported yet.
```

Comment:
```suggestion
The `original GPU build <#build-gpu-version>`__ of LightGBM is based on OpenCL.

The CUDA-based build is a separate implementation and requires an NVIDIA graphics card with compute capability 6.0 and higher. It should be considered experimental, and we suggest using it only when it is impossible to use OpenCL version (for example, on IBM POWER microprocessors).

**Note**: only Linux is supported, other operating systems are not supported yet.
```

When I read this, I was really confused whether "this version" refers to OpenCL or CUDA. Could we clarify it?

Reply:
Fixed in 4a175a9625fe2542abb600bcd6647927157b4dbe.